api/external/events.html

```diff
@@ -0,0 +1,530 @@
+                        <dl class="delete">
+                            <dt id="delete--#account_id-member_event_type_by_id">
+                                <tt class="descname">Delete </tt>
+                                <tt class="descname">/#account_id/events/#event_type_id</tt>
+                                <a class="headerlink" href="#delete--#account_id-member_event_type_by_id" title="Permalink to this definition">&para;</a>
+                            </dt>
+                            <dd>
+                                <p>Delete a specific member event type.</p>
+                                <table class="docutils field-list" frame="void" rules="none">
+                                    <col class="field-name" />
+                                    <col class="field-body" />
+                                    <tbody valign="top">
+                                    <tr class="field-even field">
+                                        <th class="field-name">Raises:</th>
+                                        <td class="field-body">
+                                            <tt class="docutils literal">
+                                                <span class="pre">Http404</span>
+                                            </tt>
+                                            if no event type with the specified ID is found.
+                                        </td>
+                                    </tr>
+                                    </tbody>
+                                </table>
+                            </dd>
+                        </dl>
+                        <p>With these endpoints, you can create member events and view information about specific member's events.</p>
+                        <dl class="post">
+                            <dt id="post--#account_id-member_event">
+                                <tt class="descname">POST </tt>
+                                <tt class="descname">/#account_id/events/members</tt>
+                                <a class="headerlink" href="#post--#account_id-member_event" title="Permalink to this definition">&para;</a>
+                            </dt>
+                            <dd>
+                                <p>Create up to 10 member events.</p>
+                                <table class="docutils field-list" frame="void" rules="none">
+                                    <col class="field-name" />
+                                    <col class="field-body" />
+                                    <tbody valign="top">
+                                    <tr class="field-odd field"><th class="field-name">Parameters:</th>
+                                        <td class="field-body">
+                                            <ul class="first simple">
+                                                <li><strong>events</strong> (<em>array of dictionaries</em>) &#8211; Required. A list of member event dictionaries.
```

Comment:
A dictionary is specific to Python. We can instead specify a JSON object, as that's less specific to Python.

```suggestion
                                                <li><strong>events</strong> (<em>array of JSON objects</em>) &#8211; Required. A list of member event JSON objects.
```